app/controllers/books_controller.rb

```diff
@@ -6,7 +6,7 @@ class BooksController < ApplicationController
   # GET /books
   # GET /books.json
   def index
-    @books = Book.all
+    @books = Book.order(created_at: :desc).order(:id).page(params[:page])
```

Comment:
結果はOKあってるのですが、下記の方がわかりやすいかもです。

```suggestion
    @books = Book.order(created_at: :desc, id: :asc).page(params[:page])
```

Reply:
並びを一意にするためのidの指定の仕方がわからず、勉強になりました。
ご指摘ありがとうございます。